store/store.go

```diff
@@ -424,6 +428,12 @@ func (bs *BlockStore) SaveSeenCommit(height int64, seenCommit *types.Commit) err
 	return bs.db.Set(calcSeenCommitKey(height), seenCommitBytes)
 }
 
+// IpfsAPI returns the ipfs api object of the BlockStore. Fullfills the
+// state.BlockStore interface.
+func (bs *BlockStore) IpfsDagAPI() format.DAGService {
```

Comment:
What about only `DAG()` or `IpfsDAG()`? Api seems redundant to me here 

Also, the comment does not correspond to the name

Reply:
removed the entire method https://github.com/lazyledger/lazyledger-core/pull/356/commits/8a4026c912c8e60eca5070d92c0a0a41fd30147d because we were no longer using it after fixing https://github.com/lazyledger/lazyledger-core/pull/356#discussion_r640817172